snap/hooks/connect-plug-etc-opt-chrome-native-messaging-jabref

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [ ! -d /var/lib/snapd/hostfs/usr/lib/mozilla/native-messaging-hosts ]; then
+    echo "Missing directory, create it manually then try again:"
+    echo "sudo mkdir -p /etc/opt/chrome/native-messaging-hosts"
+    exit 1
+fi
+
+cp $SNAP/lib/native-messaging-host/chromium/org.jabref.jabref.json /etc/opt/chrome/native-messaging-hosts/org.jabref.jabref.json
```

Comment:
Please fix that "No newline at end of file" here and in the other files. Rest is fine!

Reply:
Done!